### ISSUE TITLE
Command Key 

### ISSUE THREAD

**Issue by matthewgithub** [webBoxio/atom-bootstrap3#2]
What is the command key to toggle completion?

The README defines triggers; however how do you 'trigger' one of the triggers?
## I want to type:

page-header 
### And Get:

a div with < div class="page-header" >
a heading 1 and a small 
with a < / div > closed.

With Sublime Text 3 you do CTRL + SHIFT + P and just type 'bs3'

**Comment by bmoneruiux** [comment]
Ok. This conflicts with Emmet, which uses TAB for expansion. Can these two live in harmony, or no?

**Comment by aissarmurad** [comment]
I have a same problem with TAB for expansion. This conflicts with Emmet, How to change the Atom-bootstrap3's hotkey to Ctrl+TAB?

**Comment by briantully** [comment]
no answer for this yet? does this package just not work? I do not have the Emmet package installed and yet when I press tab all I get is a tab character, no autocomplete or snippets

**Comment by partyparrotgreg** [comment]
Yup, no information of what key expands the snippet. Totally useless
